tests/containers/podman_bci_systemd.pm

```diff
@@ -30,6 +30,9 @@ sub run {
     record_info('Test', 'Start nginx');
     assert_script_run("podman exec nginx systemctl start nginx");
 
+    record_info('Test', 'Wait for container to be healthy');
+    script_retry("podman inspect -f '{{.State.Health.Status}}' nginx | grep -x healthy", retry => 10, delay => 15);
```

Comment:
Causing an issue in PC 15sp1 https://openqa.suse.de/tests/12226863#step/podman_bci_systemd/78